src/Host/Client/Impl/Host/BrokerClientProxy.cs

```diff
@@ -26,10 +26,10 @@ internal sealed class BrokerClientProxy : IBrokerClient {
             broker.Dispose();
         }
 
-        public string Name => _broker.Name;
-        public bool IsRemote => _broker.IsRemote;
-        public Uri Uri => _broker.Uri;
-        public AboutHost AboutHost => _broker.AboutHost;
+        public string Name => _broker?.Name;
```

Comment:
Sure

---
In reply to: [81586637](https://github.com/Microsoft/RTVS/pull/2416#discussion_r81586637) [](ancestors = 81586637,81585142,81584799)